packages/flutter/lib/src/services/system_chrome.dart

```diff
@@ -77,17 +78,97 @@ enum SystemUiOverlay {
   bottom,
 }
 
+/// Specifies a dark vs light preference for certain system chrome options.
+enum SystemChromeTheme {
+  /// Whether the setting should be drawn to be compatible with a light color.
+  light,
+  /// Whether the setting should be drawn to be compatible with a dark color.
+  dark,
+}
+
 /// Specifies a preference for the style of the system overlays.
 ///
 /// Used by [SystemChrome.setSystemUIOverlayStyle].
-enum SystemUiOverlayStyle {
+class SystemUiOverlayStyle {
   /// System overlays should be drawn with a light color. Intended for
   /// applications with a dark background.
-  light,
+  static const SystemUiOverlayStyle light = const SystemUiOverlayStyle(
+    navigationBarColor: 0xFF000000,
+    navigationDividerColor: null,
+    statusBarColor: null,
+    navigationIconTheme: SystemChromeTheme.light,
+    statusBarTheme: SystemChromeTheme.light,
+  );
 
   /// System overlays should be drawn with a dark color. Intended for
   /// applications with a light background.
-  dark,
+  static const SystemUiOverlayStyle dark = const SystemUiOverlayStyle(
+    navigationBarColor: 0xFFFFFFFF,
+    navigationDividerColor: null,
+    statusBarColor: null,
+    navigationIconTheme: SystemChromeTheme.dark,
+    statusBarTheme: SystemChromeTheme.dark,
+  );
+
+  /// Creates a new [SystemUiOverlayStyle].
+  const SystemUiOverlayStyle({
+    this.navigationBarColor,
+    this.navigationDividerColor,
+    this.statusBarColor,
+    this.statusBarTheme,
+    this.navigationIconTheme,
+  });
+
+  /// The color of the bottom navigation bar.
```

Comment:
Even though the doc's context informs otherwise, think it's worth qualifying this as the "system" navigation bar to avoid confusion with [BottomNavigationBar](https://docs.flutter.io/flutter/material/BottomNavigationBar-class.html).

Reply:
Done